.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: https://editorconfig.org/
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+trim_trailing_whitespace = true
```

Comment:
isn't this already defined by prettier config? i.e. you need to set up your editor to format/highlight with a prettier extension